corehq/apps/reports/filters/users.py

```diff
@@ -141,8 +142,10 @@ def user_type_tuple(self, t):
         )
 
     def location_tuple(self, location):
-        return ("l__%s" % location.location_id,
-                '%s [location]' % location.get_path_display())
+        text = location.get_path_display()
+        if self.include_suffixes:
+            text = f'{text} [location]'
```

Comment:
that's a weird one, where is the suffix used?

Reply:
In exports, where you have a single box that searches mobile users, web users, groups, and locations, it's useful to distinguish between them.